dandiapi/api/views/serializers.py

```diff
@@ -100,6 +100,12 @@ class Meta:
     dandiset = DandisetSerializer()
     # name = serializers.SlugRelatedField(read_only=True, slug_field='name')
 
+    def __init__(self, *args, **kwargs):
+        if kwargs.get('context', {}).get('child', False):
```

Comment:
Is there a reason to reflect into `kwargs` rather than just include `context=None` as a keyword arg?

Reply:
You raise a good point. I've reworked the implementation to pluck this kwarg out, no longer passing it to the underlying base serializer class.